src/LcdButtons.cpp

```diff
@@ -10,11 +10,11 @@ LcdButtons::LcdButtons(byte pin, LcdMenu* lcdMenu) {
     _analogPin = pin;
     _lastKeyChange = 0;
 
-    _newKey = -1;
-    _lastNewKey = -2;
+    _newKey = btnNONE;
```

Comment:
I don't think this is a functional change, but we could add `btnINVALID` if this could cause issues.